package.json

```diff
@@ -235,6 +237,46 @@
       "@ember/controller/index.js": "ember-source/@ember/controller/index.js",
       "@ember/debug/container-debug-adapter.js": "ember-source/@ember/debug/container-debug-adapter.js",
       "@ember/debug/data-adapter.js": "ember-source/@ember/debug/data-adapter.js",
+      "@ember/debug/ember-inspector-support/adapters/basic.js": "ember-source/@ember/debug/ember-inspector-support/adapters/basic.js",
```

Comment:
do all of these need to be separate entries here? or can we bundle it all under ember-inspector-support.js?

Reply:
I'm not sure if it was me or some automated task that did this. Probably not necessary to have all here. Should never be directly imported by client code